tests/unittests/test_handler/test_schema.py

```diff
@@ -93,7 +130,7 @@ def test_validateconfig_schema_emits_warning_on_missing_jsonschema(self):
         with mock.patch.dict('sys.modules', **{'jsonschema': ImportError()}):
             validate_cloudconfig_schema({'p1': -1}, schema, strict=True)
         self.assertIn(
-            'Ignoring schema validation. python-jsonschema is not present',
```

Comment:
While I'm making changes, change this to python package name, rather than distro-specific name.